src/pages/components/modal/accessibility.mdx

```diff
@@ -0,0 +1,381 @@
+---
+title: Modal
+description: Modals communicate information via a secondary window and allow the user to maintain the context of a particular task. Modals interrupt user workflow by design.
+tabs: ['Code', 'Usage', 'Style', 'Accessibility']
+---
+
+import {
+  StructuredListWrapper,
+  StructuredListHead,
+  StructuredListBody,
+  StructuredListRow,
+  StructuredListInput,
+  StructuredListCell,
+  OrderedList,
+  ListItem,
+} from 'carbon-components-react';
+
+<AnchorLinks>
+  <AnchorLink>How it works</AnchorLink>
+  <AnchorLink>Accessibility considerations</AnchorLink>
+  <AnchorLink>Resources</AnchorLink>
+  <AnchorLink>Accessibility testing</AnchorLink>
+</AnchorLinks>
+
+## How it works
+
+The Modal Carbon component is a secondary window that displays over the primary window to allow the user to maintain the context of a particular task. When the modal is displayed the primary window is inert, so users cannot interact with content outside an active modal.
+
+The modal dialog contains the tab sequence so that `Tab` and `Shift-Tab` keys do not move the focus outside of the modal. To support screen reader users each Modal Component includes a WAI ARIA `role="dialog"` and `aria-modal="true"` so assistive technologies understand that the windows underneath the current dialog are not available for interaction (inert). Focus, accessible descriptions, and labels are set based on the content of each Modal. Modals that contain static text have the `aria-describeby` property set on the element with the ARIA `role="dialog"` to indicate which element or elements in the dialog contain content describes the primary purpose or message of the dialog. A value is also set for the `aria-labelledby` property that refers to a visible dialog title.
```

Comment:
```suggestion
The modal dialog contains the tab sequence so that `Tab` and `Shift-Tab` keys do not move the focus outside of the modal. To support screen reader users each modal component includes a WAI ARIA `role="dialog"` and `aria-modal="true"` so assistive technologies understand that the windows underneath the current dialog are not available for interaction (inert). Focus, accessible descriptions, and labels are set based on the content of each modal. Modals that contain static text have the `aria-describeby` property set on the element with the ARIA `role="dialog"` to indicate which element or elements in the dialog contain content describes the primary purpose or message of the dialog. A value is also set for the `aria-labelledby` property that refers to a visible dialog title.
```